static/app/components/events/interfaces/frame/packageLink.tsx

```diff
@@ -19,52 +18,47 @@ type Props = {
   isHoverPreviewed?: boolean;
 };
 
-class PackageLink extends Component<Props> {
-  handleClick = (event: React.MouseEvent<HTMLAnchorElement>) => {
-    const {isClickable, onClick} = this.props;
-
+function PackageLink({
+  children,
+  includeSystemFrames,
+  isClickable,
+  isHoverPreviewed,
+  onClick,
+  packagePath,
+  withLeadHint,
+}) {
```

Comment:
shall we use `Props` here?
```suggestion
}: Props) {
```

Reply:
whoops thanks!